src/App.tsx

```diff
@@ -1,172 +1,25 @@
-import { Loader } from './components/Loader';
-
+import { Routes, Route, Navigate } from 'react-router-dom';
 import './App.scss';
+import { Navbar } from './components/Navbar';
+import { HomePage } from './pages/HomePage';
+import { PeoplePage } from './pages/PeoplePage';
+import { NotFoundPage } from './pages/NotFoundPage';
```

Comment:
create `index.ts` file in the `pages` folder, export each of your component, and use it here like
```
import { HomePage, PeoplePage, NotFoundPage } from './pages'
```